generator/TypesGenerator.ts

```diff
@@ -0,0 +1,249 @@
+import fs from 'fs';
+import path from 'path';
+import directoryTree from 'directory-tree';
+import { Project, Writers } from 'ts-morph';
+import {
+  quicktype, InputData, JSONSchemaInput, FetchingJSONSchemaStore,
+} from 'quicktype-core';
+
+import {
+  apiMapFile, schemaFolder, typesLabel, typeFile,
+} from './Utils.js';
+
+console.log(schemaFolder);
+if (!fs.existsSync(schemaFolder)) {
+  console.log('Schemas not found, please clone the "PokeAPI/api-data" repository on the root of this project first and change all $refs from "/schema/v2" to "api-data/data/schema/v2")');
```

Comment:
We should then add an `npm` script that does the cloning and the replacing. How did you do the replacing? With `sed`?

Reply:
VSCode, a global replace